lib/rules/maxLength.js

```diff
@@ -1,5 +1,19 @@
 var _ = require('../helper');
 
+/**
+ * Returns the byte length of a string
+ * @param {string} str 
+ * @returns {number}
+ */
+function getByteLength(str) {
+  if (!str) return 0;
+  if (typeof TextEncoder !== 'undefined') {
+    return new TextEncoder().encode(str).length;
+  }
+  // Fallback for very old browsers
+  return new Blob([str]).size;
```

Comment:
Should we still prefer `Buffer.byteLength` if available? How does the overhead from these approaches compare? Are they all the same?